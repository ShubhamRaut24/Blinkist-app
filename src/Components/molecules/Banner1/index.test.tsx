import React from 'react';
import Banner1 from ".";

import renderer from 'react-test-renderer';


test('Banner', () => {
    const component = renderer.create(
            <Banner1></Banner1>
    );
    expect(component).toMatchSnapshot();
});