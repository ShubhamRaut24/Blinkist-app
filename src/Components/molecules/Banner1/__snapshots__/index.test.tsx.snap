// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Banner 1`] = `
<div
  className="MuiGrid-root MuiGrid-container"
  style={
    Object {
      "background": "#F1F6F4",
      "height": "270px",
      "position": "relative",
      "width": "980px",
    }
  }
>
  <div
    className="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-direction-xs-column MuiGrid-wrap-xs-nowrap"
    style={
      Object {
        "justifyContent": "center",
        "width": "618px",
      }
    }
  >
    <div
      className="MuiGrid-root MuiGrid-item"
      style={
        Object {
          "height": "90px",
          "margin": "45px 548px 129px 45px",
          "position": "absolute",
          "width": "319px",
        }
      }
    >
      <h3
        className="MuiTypography-root MuiTypography-h3 css-124w0ol-MuiTypography-root"
        variant1="h3"
      >
        Explore Books on entrepreneurship
      </h3>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item"
      style={
        Object {
          "height": "69px",
          "margin": "150px 406px 45px 45px",
          "position": "absolute",
          "width": "461px",
        }
      }
    >
      <h6
        className="MuiTypography-root MuiTypography-subtitle2 css-b163ci-MuiTypography-root"
        variant1="subtitle2"
      >
        Everything you need to know about thriving on a
        <br />
         shoestring budget, making your first million, and hiring
        <br />
         right from the start
      </h6>
    </div>
  </div>
  <div
    className="MuiGrid-root MuiGrid-item"
    style={
      Object {
        "width": "294px",
      }
    }
  >
    <img
      alt="not"
      height="230px"
      src="exploreposter.png"
      width="249px"
    />
  </div>
</div>
`;
